fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -224,11 +225,17 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
                 )!!,
                 contentDescription = context.getString(R.string.review_quality_check_open_handle_content_description),
                 iconTintColorResource = ThemeManager.resolveAttribute(R.attr.textPrimary, context),
-                visible = { reviewQualityCheckAvailable },
+                visible = {
+                    if (reviewQualityCheckAvailable) {
+                        Shopping.addressBarIconDisplayed.record()
```

Comment:
I think visible can be called at different times based on how this view invalidates, we should set this in `ReviewQualityCheckFeature`. Can also be tested that called once per product page tab.

Reply:
This was indeed called multiple times per product page tab. I updated the event to be recorded only when flipping `reviewQualityCheckAvailable` from `false` to `true`. Should we still move it to `ReviewQualityCheckFeature`?

Reply:
This seems good 👍